### ISSUE TITLE
Manual in wiki format

### ISSUE THREAD

**Issue by fedy2** [ieeecs/seradio-manual#1]
Would be nice to have the manual in wiki format so other people can contribute to it. The wiki format would help also the translation to other languages.

**Comment by RobertBlumen** [comment]
@fedy2 I see the benefits of this idea.  By wiki format do you mean one of the existing wiki markups such as wikitext or the format used by MediaWiki?  Or do you mean a hosted wiki site with the manual as a page on the site?  Currently I am not aware that the computer society has a hosted wiki that could be opened to the public.

**Comment by RobertBlumen** [comment]
@fedy2 second question - The source of the document is now a Google doc, which changes on a monthly basis.  I have done conversions from an office-type doc to a markup language before, never with very good results.  There is usually an automated step that does 70% and 30% manual cleanup required.  I wouldn't mind doing this once but from that point forward I really have two sources that would need to be maintained in parallel.  If there is a fully automated reliable and accurate conversion then I could do the conversion a few times per year and merge the diffs.  Do you have experience with any automated and accurate conversions from Google docs to wiki markup?

**Comment by fedy2** [comment]
Yes I mean markdown format. It is supported by GitHub and can be edited directly on the GitHub site. Is the same format used for the readme file in the repository. Here a similar project: https://github.com/johnpapa/angular-styleguide

**Comment by fedy2** [comment]
About the second question: I don't know any automated tool for conversion between Google doc and markdown. The idea of using the markdown format is a way to let other people collaborate to the manual using the git repository in GitHub. I think would be complex to keep a separate google doc copy where you have to copy the users suggestions/modifications.

**Comment by RobertBlumen** [comment]
ok - within the next few months, or, when I can get to it, I will make an attempt to export the google doc to a markdown.

**Comment by molk** [comment]
please consider [asciidoc](http://asciidoctor.org/) instead of markdown.
please ...
